draft-ietf-masque-connect-udp-listen.md

```diff
@@ -213,8 +215,12 @@ COMPRESSION_CLOSE with the context ID (see {{capsulecloseformat}}) if it
 doesn't wish to support compression for the given Context ID (For example, due
 to the memory cost of establishing a list of mappings per target per client).
 If the compression was rejected, the client and proxy will instead use an
-uncompressed context ID (See {{uncompressed}}) to exhange UDP payloads for the
-given target, if those have been enabled.
+uncompressed context ID (See {{uncompressed}}) to exchange UDP payloads for the
+given target, if those have been enabled. Only one Context ID can be used per
+IP-port tuple. If an endpoint detects that both itself and its peer have opened a context ID for the same tuple, the endpoint MUST close the context ID that was opened by the server. If
+a peer attempts to allocate another Context ID for a tuple which already has
```

Comment:
The difference between "that" and "which" is that if it's "which" then you can remove the "which ..." clause and the sentence still has the same meaning

```suggestion
a peer attempts to allocate another Context ID for a tuple that already has
```